content/contribute/helping-out/reporting-issues.md

```diff
@@ -5,9 +5,9 @@ insert_anchor_links = "right"
 weight = 2
 +++
 
-Bevy tracks bugs and feature requests using [Github Issues], which is an issue tracker and discussion space. You can find Bevy's issue page [here](https://github.com/bevyengine/bevy/issues), where you can navigate, filter, and view individual issues.
+Bevy tracks bugs and feature requests using [GitHub Issues], which is an issue tracker and discussion space. You can find Bevy's issue page [here](https://github.com/bevyengine/bevy/issues), and the Bevy Website's issue page [here](https://github.com/bevyengine/bevy-website/issues), where you can navigate, filter, and view individual issues.
```

Comment:
```suggestion
Bevy tracks bugs and feature requests using [GitHub Issues], which is an issue tracker and discussion space. You can find Bevy's issue page [here](https://github.com/bevyengine/bevy/issues), and Bevy's website's issue page [here](https://github.com/bevyengine/bevy-website/issues), where you can navigate, filter, and view individual issues.
```

I don't like "the" in this sentence, nor how "Website" is capitalized. I don't really treat the website as its own _thing_, but instead something that accompanies Bevy.